src/c/weaver/CMakeLists.txt

```diff
@@ -1,5 +1,5 @@
 # support C++14 features used by LLVM 10.0.0
-set(CMAKE_CXX_STANDARD 14)
+set(CMAKE_CXX_STANDARD 17)
```

Comment:
This should be determined based on the LLVM version used to compile the plugin. Basically, use whatever was the development default in that LLVM version.